src/super_gradients/common/factories/transforms_factory.py

```diff
@@ -31,8 +36,12 @@ def __init__(self):
             "DetectionHorizontalFlip": DetectionHorizontalFlip,
             "DetectionPaddedRescale": DetectionPaddedRescale,
             "DetectionTargetsFormat": DetectionTargetsFormat,
-            "DetectionTargetsFormatTransform": DetectionTargetsFormatTransform
+            "DetectionTargetsFormatTransform": DetectionTargetsFormatTransform,
 
+            'RandomResizedCropAndInterpolation': RandomResizedCropAndInterpolation,
+            'RandomAugmentation': rand_augment_transform,
```

Comment:
This isn't just "random augmentation", the name RandAugment has a meaning in literature.
Please rename it to RandAugmentTransform.